engines/director/channel.cpp

```diff
@@ -537,24 +555,47 @@ void Channel::replaceSprite(Sprite *nextSprite) {
 	}
 }
 
+void Channel::setNonChangedSprite(Sprite *nextSprite) {
```

Comment:
What is the prurpose of it? Looks like a dirty hack or no reason. You should not check the positions like this.